### PR TITLE
Martinquenneville caption

### PR DESCRIPTION
**Pull request by martinquenneville-zz**
Added caption and title

### CODE REVIEW

File: _posts/0000-01-13-martinquenneville.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 layout: slide
-title: "Welcome to our slide deck!"
+title: "This is Martin Quenneville's slide deck!"
 ---
 
 ![chill-out-lemur](https://cloud.githubusercontent.com/assets/16547949/25400761/9c64ee82-29c1-11e7-8ad3-6886f97634b9.jpg)
 {: .center}
 
-Caption here
+Don't worry, I've got this!
```
Comment (jmbouffard) [inline _posts/0000-01-13-martinquenneville.md:9]: @martinquenneville Are you sure people will understand the 2nd degree of the joke ?

File: _posts/0000-01-13-martinquenneville.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "This is Martin Quenneville's slide deck!"
```
Comment (jmbouffard) [inline _posts/0000-01-13-martinquenneville.md:3]: @martinquenneville  Perhaps you would prefer not using your name?

**Review by jmbouffard — COMMENTED** [review]
It can be submitted once you review my code comments

File: _posts/0000-01-13-martinquenneville.md (hunk 2 above)
Comment (martinquenneville-zz) [reply _posts/0000-01-13-martinquenneville.md:3]: But i have no imagination

Comment (jmbouffard) [reply _posts/0000-01-13-martinquenneville.md:3]: OK I will approve anyway :smiley:

**Review by jmbouffard — APPROVED** [review]
Approved for merging

**Pull request merged**